### PR TITLE
break: LANZ-2766 Add baselines

### PR DESCRIPTION
**Pull request by jstehlik-ohpen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,110 @@ All metadata will be exposed in a map that can be consumed in your terraform con
...
+- Backwards compatibility must be ensured
+
+### Module inputs
+Our accounts are client-specific. The two distinguishing attributes for retrieving the correct information are
+**stage** and **deployment**.
+
+| Variable   | Description                                      | Required | Default |
+|------------|--------------------------------------------------|----------|---------|
+| stage      | Stage of the resources - e.g. dev, tst, acc, prd | yes      | N/A     |
+| deployment | Deployment in a stage - e.g. main, a, b, c       | no       | main    |
```
Comment (jstehlik-ohpen) [inline README.md]: will be expanded

File: variables.tf [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+
+variable "cloudmap" {
+  description = "Return outputs from CloudMap baseline"
+  default     = {
+    enable     = true
+    stage      = null
+    deployment = null
+  }
+}
+
```
Comment (cangulomascarell) [inline variables.tf]: as there is a risk some SSM parameters are not defined for specific services as cloudmap or ecs, maybe we can set the enable to `false` , so the user should enable only what he needs, only that should **exists** . In that way we avoid any logic to deal with unexistent parameters. What do you think guys ?

Comment (jstehlik-ohpen) [reply variables.tf]: I agree. The next question then is if all will be false by default and users will set it to true, or if we control what is true/false by default based on which baselines are deployed. First option seems more consistent to me.

File: README.md [hunk 3]
```diff
@@ -13,3 +13,110 @@ All metadata will be exposed in a map that can be consumed in your terraform con
 ## TODO: Exposed metadata
 
 ## TODO: Examples
+
```
Comment (jstehlik-ohpen) [inline README.md:35]: Read for solution reference, will be rewritten later

File: variables.tf [hunk 4]
```diff
@@ -1,5 +1,45 @@
...
   type        = string
   description = "The id of the aws account we want to extract the metadata from."
 }
+
+variable "stage" {
+  type        = string
+  description = "Stage of the outputs, e.g. dev, tst, acc, prd"
+}
+
+variable "deployment" {
```
Comment (jstehlik-ohpen) [inline variables.tf]: Will possibly be removed and deployment set per baseline - requires knowledge of which deployment is needed by baseline, but is consistent - no strange errors, you get what you ask for

**Review by cangulomascarell — DISMISSED** [review]

**Pull request closed without merging**